test/realmodel/index.html

```diff
@@ -44,15 +44,49 @@
         a.dispatchEvent(e)
       }
 
+      async function downloadFirstResnet(data, filename) {
+        if (!data) {
+          console.error("downloadjson: No data");
+          return;
+        }
+        if (!filename) filename = "casePrototypeData.json";
+        if (typeof data === "object") {
+          data_tensor = JSON.stringify(data.tensorTypes);
+          data_operation = JSON.stringify(data.operation);
+          data_operations = JSON.stringify(data.operations);
+          data = `{"operation": ${data_operation}, "tensorTypes": ${data_tensor}, "operations": ${data_operations}}`;
+        }
+        var blob = new Blob([data], {type: 'text/json'}),
+            e = document.createEvent('MouseEvents'),
+            a = document.createElement('a');
+        a.download = filename;
+        a.href = window.URL.createObjectURL(blob);
+        a.dataset.downloadurl = ['text/json', a.download, a.href].join(':');
+        e.initMouseEvent('click', true, false, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
+        a.dispatchEvent(e)
+      }
 
-      async function downloadjson(data, filename) {
+      async function downloadSeconedResnet(data, filename, number1, number2) {
```

Comment:
@shenx3x Was the world 'Seconed' clerical error? And please add explanation for **number1** / **number2** parameters, thanks